src/components/components/ebay-dialog-base/index.marko

```diff
@@ -22,8 +24,13 @@ $ const baseEl = input.baseEl || 'div';
             <${input.top.renderBody}/>
         </if>
         <${input.header ? "div" : null} class=`${input.classPrefix}__header`>
+            $ const header = input.header || {};
```

Comment:
I'd rather see `<if(input.header && buttonPosition === 'right')>` than defaulting to an empty object here. Also as it stands I believe this would create empty `<h2>`'s if you don't pass a header?